Doc/library/py_compile.rst

```diff
@@ -22,6 +22,36 @@ shared use, especially if some of the users may not have permission to write the
 byte-code cache files in the directory containing the source code.
 
 
+Command-line usage
+------------------
+
+This module can be invoked as a script to compile several source files.
+The files named in *filenames* are compiled and the resulting bytecode
+is cached in the normal manner.  This program does not search a directory
+structure to locate source files; it only compiles files named explicitly.
+
+.. program:: python -m py_compile
+
+.. cmdoption:: file ...
+               -
+
+   Positional arguments are files to compile. If ``-`` is the only
+   parameter, the list of files is taken from standard input.
+
+.. cmdoption:: -q, --quiet
+
+   Suppress errors output.
+
+.. versionchanged:: 3.2
+   Added support for ``-``.
+
+.. versionchanged:: 3.9
+   Added support for ``-q/--quiet``.
+
+
+Public functions
```

Comment:
We usually don't use "public functions" section names in the documentation. If something is documented, it's part of the public API.

It would be better if we remove the section and move the "Command-line usage" section to the bottom of the file.

Reply:
done